subsys/bluetooth/host/hci_core.c

```diff
@@ -4667,8 +4666,8 @@ static void le_per_adv_sync_lost(struct net_buf *buf)
 {
 	struct bt_hci_evt_le_per_adv_sync_lost *evt =
 		(struct bt_hci_evt_le_per_adv_sync_lost *)buf->data;
-	struct bt_le_per_adv_sync *per_adv_sync;
 	struct bt_le_per_adv_sync_term_info term_info;
+	struct bt_le_per_adv_sync *per_adv_sync;
```

Comment:
How is this change related? If it's cleanup it should be a separate commit (can still be same PR)

Reply:
Fixed.